### ISSUE TITLE
Unexpected key extensions

### ISSUE THREAD

**Issue by IchordeDionysos** [firebase/firebase-functions#1600]
### [REQUIRED] Version info

**node:** 20

**firebase-functions:** v5.1.0

**firebase-tools:** v13.15.4

**firebase-admin:** v12.4.0

### [REQUIRED] Test case

Version 5.0.1 works upgrading to 5.1.0 fails.

### [REQUIRED] Steps to reproduce

Deploy `firebase deploy` with Cloud Run Functions.

### [REQUIRED] Expected behavior

The deployment works.

### [REQUIRED] Actual behavior

When deploying using `firebase deploy`, we get the following error:

```
FirebaseError Unexpected key extensions. You may need to install a newer version of the Firebase CLI.
```

### Were you able to successfully deploy your functions?

```
FirebaseError Unexpected key extensions. You may need to install a newer version of the Firebase CLI.
```

**Comment by IchordeDionysos** [comment]
@ifielker @taeold can you take a look?
It seems like your PR causes the issues we are facing.

**Comment by MalachiBatt** [comment]
Try updating Firebase CLI (firebase-tools). Worked for me

**Comment by modernlabrat** [comment]
I am receiving this same issue. I tried uninstalling firebase-tools and reinstalling, still is not working and cannot use emulators either for functions

**Comment by IchordeDionysos** [comment]
Yeah I know that the error says that but we are using version 13.5.4

**Comment by mdulc928** [comment]
Yeah, even after installing the latest version of `firebase-tools`, I kept checking and still came up as though I was using the old version, even though when I listed the my global packages, I had the latest version installed. 

Long story short, I killed the terminal I was using and it worked magically 🤦

**Comment by Madi0ne** [comment]
updating to latest version of firebase-tools (13.16.0) fixes the issue for me.

**Comment by adamwreuben** [comment]
updating to latest version of firebase-tools (13.16.0) also fixed the issue for me.
